bin/console

```diff
@@ -48,7 +48,7 @@ if (isset($_SERVER['argv']) && ['dependencies', 'install'] === array_slice($_SER
 
    if (array_key_exists('ci', $options) && true === $options['ci']) {
       $composer_command .= ' --prefer-dist --no-progress';
-      $npm_command = 'npm ci';
+      //$npm_command = 'npm ci';
```

Comment:
You should drop or uncomment :) (dunno why you commented, nor what is the original purpose).

Reply:
Purpose was to not alter package-lock.json on install, but it is not important anymore as we changed way to check if dependencies are up to date.